recipes/skprogs/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "skprogs" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dftbplus/skprogs/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e63e6a38f920bbb85f1f8fdff2da5ec195a3f2ded354f6f09cf68cf69785b59d
+  patches:
+    - no-python.patch
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - ninja
+    - pkg-config
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - libblas
+    - liblapack
+    - libxc
```

Comment:
Please check the [build logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=519794&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2363). It may be that blas is not used or that it is being statically linked instead of dynamically linked.

Reply:
Looks like upstream is overdepending on LAPACK/BLAS. Maybe @aradi can comment on the necessity of having a linear algebra library around.